restful_api/myapi/models/course_trainer.py

```diff
@@ -0,0 +1,13 @@
+from myapi.extensions import db
+
+class CourseTrainer(db.Model):
+    """Basic course trainer model"""
+
+    __tablename__ = "course_trainer"
+    __table_args__ = (db.UniqueConstraint('course_id','trainer_id', name = 'unique_course_trainer'),)
```

Comment:
no problems here, just explicitly noting for BE folks @joelytj , I think some of our `models` aren't following this convention, let's standardise to this